cmd/flags/prover.go

```diff
@@ -34,153 +30,179 @@ var (
 		Name:     "raiko.hostEndpoint",
 		Usage:    "RPC endpoint of a Raiko host service",
 		Category: proverCategory,
+		EnvVars:  []string{"RAIKO_HOST_ENDPOINT"},
 	}
 	RaikoL1Endpoint = &cli.StringFlag{
 		Name:     "raiko.l1Endpoint",
 		Usage:    "L1 RPC endpoint which will be sent to the Raiko service",
 		Category: proverCategory,
+		EnvVars:  []string{"RAIKO_L1_ENDPOINT"},
 	}
 	RaikoL1BeaconEndpoint = &cli.StringFlag{
 		Name:     "raiko.l1BeaconEndpoint",
 		Usage:    "L1 beacon RPC endpoint which will be sent to the Raiko service",
 		Category: proverCategory,
+		EnvVars:  []string{"RAIKO_L1_BEACON_ENDPOINT"},
 	}
 	RaikoL2Endpoint = &cli.StringFlag{
 		Name:     "raiko.l2Endpoint",
 		Usage:    "L2 RPC endpoint which will be sent to the Raiko service",
 		Category: proverCategory,
+		EnvVars:  []string{"RAIKO_L2_ENDPOINT"},
 	}
 	StartingBlockID = &cli.Uint64Flag{
 		Name:     "prover.startingBlockID",
 		Usage:    "If set, prover will start proving blocks from the block with this ID",
 		Category: proverCategory,
+		EnvVars:  []string{"STARTING_BLOCK_ID"},
 	}
 	Graffiti = &cli.StringFlag{
 		Name:     "prover.graffiti",
 		Usage:    "When string is passed, adds additional graffiti info to proof evidence",
 		Category: proverCategory,
 		Value:    "",
+		EnvVars:  []string{"GRAFFITI"},
 	}
 	// Proving strategy.
 	ProveUnassignedBlocks = &cli.BoolFlag{
 		Name:     "prover.proveUnassignedBlocks",
 		Usage:    "Whether you want to prove unassigned blocks, or only work on assigned proofs",
 		Category: proverCategory,
 		Value:    false,
+		EnvVars:  []string{"PROVE_UNASSIGNED_BLOCKS"},
 	}
-	MinEthBalance = &cli.Uint64Flag{
+	MinEthBalance = &cli.Float64Flag{
 		Name:     "prover.minEthBalance",
-		Usage:    "Minimum ETH balance (in wei) a prover wants to keep",
+		Usage:    "Minimum ETH balance (in Ether) a prover wants to keep",
 		Category: proverCategory,
 		Value:    0,
+		EnvVars:  []string{"MIN_ETH_BALANCE"},
 	}
-	MinTaikoTokenBalance = &cli.Uint64Flag{
+	MinTaikoTokenBalance = &cli.Float64Flag{
 		Name:     "prover.minTaikoTokenBalance",
-		Usage:    "Minimum Taiko token balance a prover wants to keep",
+		Usage:    "Minimum Taiko token balance (in Ether) a prover wants to keep",
```

Comment:
```suggestion
		Usage:    "Minimum amount of Taiko token balance a prover wants to keep",
```
Since its just ERC20 tokens, lets just say amount here?